packages/system-tests/scenarios/integratedTerminal.test.ts

```diff
@@ -44,7 +44,7 @@ describe(TITLE, () => {
   it('Should open terminal and check that new terminals will have SFDX_SET_CLIENT_IDS', async () => {
     // Show terminal
     await app.command('workbench.action.quickOpen');
-    await common.type('>View: Toggle Integrated Terminal');
+    await common.type('>Terminal: Focus Terminal');
```

Comment:
Switch to focus command which does set cursor focus to the terminal.

Reply:
@lcampos - Did you find out if there is an existing bug on the VS Code side?
Either way, what we are doing is kosher and can be used even if they fix that regression.

Reply:
@vazexqi https://github.com/Microsoft/vscode/issues/40416 this one is the closest I found to what I see happening in our tests (start vscode, focus is on another element, open terminal and focus does not go to the terminal).